src/main/kotlin/com/cosmotech/api/rbac/CsmRbac.kt

```diff
@@ -190,22 +192,22 @@ open class CsmRbac(
 
   internal fun verifyPermissionFromRole(
       permission: String,
-      role: String,
+      role: String?,
       rolesDefinition: RolesDefinition
   ): Boolean {
     return this.verifyPermission(
         permission, this.getRolePermissions(role, rolesDefinition.permissions))
   }
 
   internal fun getRolePermissions(
-      role: String,
+      role: String?,
       rolesDefinition: MutableMap<String, List<String>>
   ): List<String> {
     return rolesDefinition[role] ?: listOf()
   }
 
-  internal fun getUserRole(rbacSecurity: RbacSecurity, user: String): String {
-    return rbacSecurity.accessControlList.firstOrNull { it.id == user }?.role ?: ROLE_NONE
+  internal fun getUserRole(rbacSecurity: RbacSecurity, user: String): String? {
+    return rbacSecurity.accessControlList.firstOrNull { it.id == user }?.role
```

Comment:
Here if the user is not defined in ACL, the fallback right should be the default one a.k.a : `rbacSecurity.default`

Reply:
Discussed & resolved by call